cmd/omegabrr/main.go

```diff
@@ -60,6 +65,19 @@ func main() {
 	switch cmd := pflag.Arg(0); cmd {
 	case "version":
 		fmt.Fprintf(flag.CommandLine.Output(), "Version: %v\nCommit: %v\n", version, commit)
+
+		// get the latest release tag from brr-api
+		resp, err := netHTTP.Get(fmt.Sprintf("https://api.autobrr.com/repos/%s/%s/releases/latest", owner, repo))
```

Comment:
What happens to the body when the status isn't ok?

Reply:
Made some improvements.